src/modules/organizations/containers/organization-container.jsx

```diff
@@ -22,13 +23,25 @@ class OrganizationContainer extends React.Component {
     this.props.dispatch(setCurrentOrganization(null));
   }
 
+  updateOrganization(organizationFields) {
+    organizationFields.name = organizationFields.display_name; // eslint-disable-line no-param-reassign
+    organizationFields.title = organizationFields.name.replace(/\W/g, ''); // eslint-disable-line no-param-reassign
+    apiClient.type('organizations').get(this.props.organization.id)
```

Comment:
Not that I know of. You could put `this.props.organization` into a new variable then call update and save on that? Then trigger a dispatch to fetch the updated resource from the API and set the new state on the store? I get that you don't want to mutate the props directly, but unnecessarily calling the API for a resource that hasn't changed yet is a current issue on PFE and one we can avoid from the start here.